nixos/modules/services/misc/home-assistant.nix

```diff
@@ -48,10 +62,22 @@ in {
       '';
       description = ''
         Home Assistant package to use.
-        Most Home Assistant components require additional dependencies,
-        which are best specified by overriding <literal>pkgs.home-assistant</literal>.
-        You can find the dependencies by searching for failed imports in your log or by looking at this list:
-        <link xlink:href="https://github.com/home-assistant/home-assistant/blob/master/requirements_all.txt"/>
+        Override <literal>extraPackages</literal> in order to add additional dependencies.
+      '';
+    };
+
+    autoExtraComponents = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        If set to <literal>true</literal>, the components used in <literal>config</config>
```

Comment:
Just fyi this line broke building manpages because of the incorrect closing of `<literal>`.

Fixed in 7ebb82e04f17d96313f1d60c748f32a7f7bee9f6

Reply:
Sorry. Thanks for fixing!